src/features/campaigns/components/ActivitiesOverview/items/CanvassAssignmentOverviewListItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { Map } from '@mui/icons-material';
+
+import { CanvassAssignmentActivity } from 'features/campaigns/types';
+import getStatusColor from 'features/campaigns/utils/getStatusColor';
+import OverviewListItem from './OverviewListItem';
+
+type Props = {
+  activity: CanvassAssignmentActivity;
+  focusDate: Date | null;
+};
+
+const CanvassAssignmentOverviewListItem: FC<Props> = ({
+  activity,
+  focusDate,
+}) => {
+  const assignment = activity.data;
+
+  return (
+    <OverviewListItem
+      color={getStatusColor(activity.visibleFrom, activity.visibleUntil)}
+      endDate={activity.visibleUntil}
+      endNumber={''}
+      focusDate={focusDate}
+      href={`/organize/${assignment.organization.id}/projects/${
+        assignment.campaign?.id ?? 'standalone'
+      }/canvassassignments/${assignment.id}`}
+      PrimaryIcon={Map}
+      SecondaryIcon={null}
+      startDate={activity.visibleFrom}
+      statusBar={null}
+      title={assignment.title || ''}
```

Comment:
I think this should not default to empty string but to `messages.canvassAssignment.empty.title` to get "Untitled canvass assignment" instead of nothing, when the title of the canvass assignment is `null`.

Reply:
Yeah we have to discuss how this should work. It varies between different kinds of activities. Events are allowed to have `null` titles, and a default title is rendered on the frontend when that occurs. Call assignments and surveys on the other hand are not allowed to have `null` titles, so their "default titles" are added when they are created instead. This could work either way (but I would lean towards the latter, in which case `title` will never be falsy).